Alert/Alert.css

```diff
@@ -0,0 +1,134 @@
+@import "../../utility/component.css";
+
+.alert{
+    position: relative;
+    padding: 1rem 1rem;
+    margin-bottom: 1rem;
+    border: 1px solid transparent;
+    border-radius: 0.25rem;
+}
+
+/*---alert example---*/
+
+#show_alert{
+    display: none;
+}
+
+#alert_example{
+    display: none;
+    position: absolute;
+    z-index: 1;
+    right: 0;
+    z-index: 999;
```

Comment:
Z-Index, not a problem in this 999 but better to use sequential like 5-6 etc. Because as project gets bigger these random big indices confuses a lot. And for other team members who might be working on with you on some project too it's not good. I read it in a blog. Give it a thought.

Reply:
ok. i will change this. Thank you so much for pointing this.

Reply:
removed z-index